datasource/src/test/scala/quasar/physical/mongo/MongoSpec.scala

```diff
@@ -128,11 +138,23 @@ object MongoSpec {
   val nonexistentDbs = List("Z", "Y")
   val nonexistentCols = List("z", "y")
 
-  lazy val connectionString = Source.fromFile("./datasource/src/test/resources/mongo-connection").mkString.trim
+  val connectionString: String = "mongodb://root:secret@127.0.0.1:27018"
+  val aConnectionString: String = "mongodb://aUser:aPassword@127.0.0.1:27018/A.a"
+  val invalidAConnectionString: String = "mongodb://aUser:aPassword@127.0.0.1:27018"
+  val bConnectionString: String = "mongodb://bUser:bPassword@127.0.0.1:27018/B.b"
```

Comment:
also worth to test with a db only connection string (like `mongodb://bUser:bPassword@127.0.0.1:27018/B`) I think